src/timeout.js

```diff
@@ -19,8 +20,7 @@ class Timeout {
 
 	rejected (p) {
 		clearTimeout(this.timer)
-		this.promise._become(p)
-		return false
+		return super.rejected(p)
```

Comment:
Probably `Action.prototype.rejected.call(this, p)`, I'm pretty sure it doesn't use a `getPrototypeOf` lookup.
`super` in `fulfilled` might be appropriate, dunno about performance, it's not saving lines though :-)